lib/tools/adb-commands.js

```diff
@@ -400,19 +404,23 @@ methods.toggleGPSLocationProvider = async function toggleGPSLocationProvider (en
  *        or to a restricted greylist for your target API level.
  */
 methods.setHiddenApiPolicy = async function setHiddenApiPolicy (value) {
-  await this.setSetting('global', 'hidden_api_policy_pre_p_apps', value);
-  await this.setSetting('global', 'hidden_api_policy_p_apps', value);
-  await this.setSetting('global', 'hidden_api_policy', value);
+  try {
+    await this.shell(HIDDEN_API_POLICY_KEYS.map((k) => `settings put global ${k} ${value}`).join(';'));
+  } catch (e) {
+    log.info(`Failed to set hidden policy keys '${HIDDEN_API_POLICY_KEYS}' to '${value}'. Original error: ${e.message}`);
```

Comment:
I'd rather say `to set setting keys`, since the other is anyway visible from their names ;)

Reply:
ah, yeah